sql/addElective.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+include "validation.php";
+
+$valid = array();
+$errors = array();
+
+$logfile = fopen("logfile.txt", "w");
```

Comment:
Why do you need to write in a text file?

Reply:
Just so I can save the log from all the operations somewhere.